nrf_rpc/include/nrf_rpc.h

```diff
@@ -28,6 +28,14 @@ extern "C" {
 /* Internal definition used in the macros. */
 #define _NRF_RPC_HEADER_SIZE 4
 
+#ifndef NRF_RPC_TR_MAX_HEADER_SIZE
+#define NRF_RPC_TR_MAX_HEADER_SIZE 0
```

Comment:
Don't define them here. Transport header file is responsible for defining it.
The same for `NRF_RPC_TR_AUTO_FREE_RX_BUF`.

Reply:
This is ifdef def, so if transport have not defined anything the values of macros are set to default.